public/blog/testrpc-is-now-ganache-cli.md

```diff
@@ -0,0 +1,25 @@
+# TestRPC is now Ganache CLI
+
+Have you been looking for the TestRPC recently and haven't been able to find anything about it?
+
+Don't worry, it's still here! It's just now known as **Ganache CLI**. You can get it here:
+
+```shell
+npm install -g ganache-cli
+```
+
+## A little history
+
+Prior to the existence of the TestRPC, one would have to deploy to a test network like Ropsten in order to see how your contract operates. The TestRPC solved that issue by providing a local blockchain where you could test your smart contracts without even needing to wait for your transactions to be mined into a block. 
```

Comment:
> blockchain where you could test your smart contracts 

Can you make this `develop and test`?

Reply:
Can you also change `network like Ropsten` with `private test networks`, or something similar? The TestRPC (Python version) existed before the mainnet was even live. Maybe something like this:

> Prior to the existence of the TestRPC, one would have to deploy to private internal test networks to see how your contract code operates (public test networks like Ropsten didn't even exist!).

Reply:
Artistic: you used a reference to `one`, as in `one would have`, but directly referenced `your` in `your contract operates`. I'd choose either or; either `one` and `their` or `you'd` and `your`.  (Note: I didn't make a selection in my example sentence above)

Reply:
Great backstory, thanks, this is the stuff I didn't know.